product/src/routes/index.js

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import categorias from "./categoriesRoutes.js"
+import produtos from "./productsRoutes.js"
+
+const routes = (app) => {
+    app.route('/').get((req, res) => {
+        res.status(200).send({titulo: "Ecomm Categories"})
+    })
```

Comment:
do you need this route?

Reply:
not really, I just put it on to test and forgot to take it off